### PR TITLE
Option Panel UI fixes and userflow

### PR DESCRIPTION
**Pull request by m4rtint**

### CODE REVIEW

**Comment by m4rtint** [comment]
[Option panel (circle slider)](https://trello.com/c/v9PnQQEp/584-option-panel-circle-slider)

**Comment by m4rtint** [comment]
[Option Panel: Volume changes applied without needing to press Save](https://trello.com/c/BXBfjtus/593-option-panel-volume-changes-applied-without-needing-to-press-save)

**Comment by m4rtint** [comment]
@mikeslgoh possible

**Comment by m4rtint** [comment]
Added Tests

**Comment by mikeslgoh** [comment]
thanks for adding in tests

File: Assets/_PartyParkade/Script/UI/MainMenu/OptionsPanel/OptionPanel.cs [hunk 1]
```diff
@@ -0,0 +1,139 @@
+﻿using _PartyParkade.Script.Data;
```
Comment (Gursaanj) [inline Assets/_PartyParkade/Script/UI/MainMenu/OptionsPanel/OptionPanel.cs:1]: missing header

File: Assets/_PartyParkade/Tests/OptionPanel_Tests.cs [hunk 2]
```diff
@@ -0,0 +1,100 @@
+﻿
```
Comment (Gursaanj) [inline Assets/_PartyParkade/Tests/OptionPanel_Tests.cs:1]: Missing Header

Comment (m4rtint) [reply Assets/_PartyParkade/Tests/OptionPanel_Tests.cs]: Added

File: Assets/_PartyParkade/Script/UI/MainMenu/OptionsPanel/OptionPanel.cs (hunk 1 above)
Comment (m4rtint) [reply Assets/_PartyParkade/Script/UI/MainMenu/OptionsPanel/OptionPanel.cs]: Added

**Pull request merged**